connectivity/FEATURE_BLE/mbed_lib.json

```diff
@@ -93,10 +93,25 @@
             "value": 15,
             "macro_name": "BLE_GAP_MAX_ADVERTISING_SETS"
         },
+        "ble-gap-host-based-privacy": {
+            "help": "Perform address resolution on the host, not the controller. Controller based privacy is preferred as it happens lower down the stack but this can be used in case controller based privacy is unavailable. If this is enabled the controller will not be used for privacy.",
+            "value": false,
```

Comment:
I'd expect this value to be on by default.

Reply:
changed to true